libs/trustchain/src/__tests__/integration/mock.sdk.test.ts

```diff
@@ -13,7 +15,7 @@ const nonMockableScenarios = [
   "userRefusesAuth", // can't simulate device interaction at the moment
 ];
 
-const scenarioFolder = path.join(__dirname, "./test-scenarios");
+const scenarioFolder = path.join(__dirname, "../../../tests/scenarios");
```

Comment:
Maybe we can add aliases for mocks and tests folder, wdyt?

Reply:
not possible with node.js path resolvers. here we will fs.read the files, it's not a require